### PR TITLE
refactor: rename `e2e::User` to `e2e::Account`

### LINKED ISSUE #138: Rename `e2e::User` to `e2e::Account`
Rationale: it's slightly more accurate and consistent with the rest of the ecosystem (`EOA`).

### PR DESCRIPTION
**Pull request by MukulKolpe**
Renamed the `e2e::User` module to `e2e::Account` maintaining consistency with the rest of the ecosystem (EOA). 

Resolves #138 

#### PR Checklist

- [x] Tests
- [x] Documentation

### CODE REVIEW

File: lib/e2e/src/user.rs
Comment (alexfertel) [inline lib/e2e/src/user.rs:1]: You would also need to rename this file `user.rs` -> `account.rs`.

Comment (MukulKolpe) [reply lib/e2e/src/user.rs:1]: Updated✅

**Review by alexfertel — APPROVED** [review]
LGTM! 🚀

**Comment by MukulKolpe** [comment]
Thanks @alexfertel, @bidzyyys!

**Pull request merged**